crypto-hash-example/filter.lua

```diff
@@ -0,0 +1,27 @@
+local digest = require 'openssl.digest'
```

Comment:
Probably want a comment to explain what things do, pre/post conditions, parameters, etc. plus a copyright header.